web-app/src/app/screens/Account.tsx

```diff
@@ -469,11 +554,9 @@ export default function APIAccount(): React.ReactElement {
                 href='https://mobilitydata.github.io/mobility-feed-api/SwaggerUI/index.html#/'
                 target={'_blank'}
               >
-                Swagger documentation
+                OpenAPI documentation
               </Link>
-              .
-              <br />
-              The access token updates regularly.
+              . Access tokens are valid for one hour.
             </Typography>
             {showGenerateAccessTokenButton && (
```

Comment:
nit: Margin is really big between the token and the curl example
<img width="1076" alt="Screenshot 2024-02-28 at 3 53 58 PM" src="https://github.com/MobilityData/mobility-feed-api/assets/90632321/7c6c012d-a388-43a3-aeb7-c1bc42bb0cc3">

Reply:
Fixed, screenshot
<img width="1348" alt="Screenshot 2024-02-28 at 4 24 25 PM" src="https://github.com/MobilityData/mobility-feed-api/assets/1192523/6c413aff-13d0-4ff7-872b-f8ac3517bd3d">